doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -225,6 +225,10 @@ Serial LTE modem
 
   * The ``AT#XPPP`` command to support the CID parameter to specify the PDN connection used for PPP.
   * The ``#XPPP`` notification to include the CID of the PDN connection used for PPP.
+  * The initialization of the application to ignore a failure in nRF Cloud module initialization.
+    This occurs sometimes especially during development.
+  * The initialization of the application to send "INIT ERROR" over to UART and show clear error log
+    to indicate that the application is not operational in case of failing initialization.
```

Comment:
```suggestion
  * The initialization of the application to send "INIT ERROR" over to UART and show clear error log to indicate that the application is not operational in case of failing initialization.
```

Reply:
Done although I would use line breaks in the middle of sentences to keep the RST readable.